rust/kernel/lib.rs

```diff
@@ -140,5 +140,50 @@ fn panic(_info: &PanicInfo) -> ! {
     }
 }
 
+/// Calculates the offset of a field from the beginning of the struct it belongs to.
```

Comment:
Could we add an example usage to this doc string?

Reply:
Done.